index.bs

```diff
@@ -247,6 +247,199 @@ a generator to a processor (e.g., with a call like
 the Web Platform-exposed signals, but ignores any internal custom signals as there is no
 way for the platform sinks to know what the upstream source is.
 
+## MediaStreamTrack ## {#generator-as-track}
+A {{MediaStreamTrackGenerator}} is a {{MediaStreamTrack}}. This section adds
+clarifications on how a {{MediaStreamTrackGenerator}} behaves as a
+{{MediaStreamTrack}}.
+
+### clone ### {#generator-clone}
+The {{MediaStreamTrack/clone}} method on a {{MediaStreamTrackGenerator}}
+returns a new {{MediaStreamTrack}} object whose underlying source is the
+same as the one for the {{MediaStreamTrackGenerator}} being cloned.
+This underlying source is the {{MediaStreamTrackGenerator/writable}} of
+the {{MediaStreamTrackGenerator}}.
+
+### stop ### {#generator-stop}
+The {{MediaStreamTrack/stop}} method on a {{MediaStreamTrackGenerator}} stops
+the track. When the last track connected to
+the {{MediaStreamTrackGenerator/writable}} of a {{MediaStreamTrackGenerator}} is
+stopped, the {{MediaStreamTrackGenerator/writable}} is closed.
+
+### Constraintable properties ### {#generator-constrainable-properties}
+
+The following constrainable properties are defined for video
+{{MediaStreamTrackGenerator}}s and any {{MediaStreamTrack}}s sourced from
+a {{MediaStreamTrackGenerator}}:
+<table>
+  <thead>
+    <tr>
+      <th>
+        Property Name
+      </th>
+      <th>
+        Values
+      </th>
+      <th>
+        Notes
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr id="def-constraint-width">
+      <td data-tests="">
+        <dfn>width</dfn>
+      </td>
+      <td>
+        {{ConstrainULong}}
+      </td>
+      <td>
+        As a setting, this is the width, in pixels, of the latest
+        frame received. As a capability, max MUST reflect the largest width
+        with which a VideoFrame object can be created, and min MUST reflect
```

Comment:
I assume "largest width" refers to the capabilities of the user agent, and has nothing to do with what the JS code is allowed to do? Similar Q for other properties.

Reply:
yes. I updated the text to say the largest width a VideoFrame may have, to make it clearer that it does not refer just to VideoFrames created by JS.